lib/src/firebase_storage_mocks_base.dart

```diff
@@ -1,51 +1,34 @@
 import 'dart:io';
-
+import 'dart:typed_data';
 import 'package:firebase_storage/firebase_storage.dart';
+import 'package:firebase_storage_mocks/src/mock_storage_reference.dart';
 import 'package:mockito/mockito.dart';
 
 class MockFirebaseStorage extends Mock implements FirebaseStorage {
-  final _ref = MockStorageReference();
+  final Map<String, File> storedFilesMap = {};
```

Comment:
When you implement "mock methods to retrieve data from a StorageReference", please make these two maps private, and rewrite the existence tests accordingly.